### PR TITLE
12 create jobadvert model

### PR DESCRIPTION
**Pull request by wojciechsi**

### CODE REVIEW

File: models/jobOfferModel.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        required: [true, "Please provide a name"],
+        maxlength: [256, 'Name too long']
+    },
+    description: {
+        type: String,
+        maxlength: [4096, 'Description is too long']
+    },
+    mode: {
+        type: [Number],
+        validate: [(val) => val.length > 0 , 'Must have minimum one type in range 1-3']
```
Comment (WojciechPtas) [inline models/jobOfferModel.js:17]: Does it really validate whether the value is in `[1,3]`?

**Review by macibol775 — APPROVED** [review]

**Review by WojciechPtas — COMMENTED** [review]
Check the comment please

File: models/jobOfferModel.js (hunk 1 above)
Comment (wojciechsi) [reply models/jobOfferModel.js:17]: Not yet, it will in the near future

File: models/jobOfferModel.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	},
+    location: {
+        type: String
+        //possibbly-TODO external locations database
+    },
+    industry: [String],
+    additionalQuestions: [String],
+    keywords: [String],
+    expireDate: Date,
+    organisationId: {
```
Comment (MasterCoookie) [inline models/jobOfferModel.js:32]: Id-related field naming convention: organization_id

File: models/jobOfferModel.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+		required: [true, 'Requirements are necessary'],
+	},
+    location: {
+        type: String
+        //possibbly-TODO external locations database
+    },
+    industry: [String],
+    additionalQuestions: [String],
+    keywords: [String],
+    expireDate: Date,
```
Comment (MasterCoookie) [inline models/jobOfferModel.js:31]: Typo: expiryDate

**Review by MasterCoookie — CHANGES_REQUESTED** [review]
Change names, I'm not usually that picky, but it's a model, so we are all gona use those

**Review by WojciechPtas — APPROVED** [review]
LGTM

**Review by MasterCoookie — APPROVED** [review]
That's a pice of the finest software

**Pull request merged**